.github/workflows/remoteconfig.yml

```diff
@@ -0,0 +1,37 @@
+name: remoteconfig
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'FirebaseRemoteConfig**'
+    - '.github/workflows/remoteconfig.yml'
+    - 'Gemfile'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+
+  quickstart:
+    env:
+      plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+      signin_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set xcode version
+      run: sudo xcode-select -switch /Applications/Xcode_11.3.app
```

Comment:
Is setting the Xcode version necessary?

Ideally I'd like to test on the GHA default - typically latest - so we catch issues early.

In the future, we could add more explicit Xcode version coverage.

Reply:
will remove the setting if the GHA default device is sufficient.